### PR TITLE
[Merged by Bors] - docs(Measure/Haar/OfBasis): expand a docstring

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean [hunk 1]
```diff
@@ -290,7 +290,18 @@ end Fintype
 /-- A finite dimensional inner product space has a canonical measure, the Lebesgue measure giving
 volume `1` to the parallelepiped spanned by any orthonormal basis. We define the measure using
 some arbitrary choice of orthonormal basis. The fact that it works with any orthonormal basis
-is proved in `orthonormalBasis.volume_parallelepiped`. -/
+is proved in `orthonormalBasis.volume_parallelepiped`.
+
+This lemma creates
```
Comment (Ruben-VandeVelde) [inline Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean:295]:
```suggestion
This instance creates
```
?

File: Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean [hunk 2]
```diff
@@ -290,7 +290,18 @@ end Fintype
...
+
+- a potential non-defeq diamond with the natural instance for `MeasureSpace (ULift E)`,
+  which does not exist in Mathlib at the moment;
+
+- the existing instance `MeasureTheory.Measure.instMeasureSpacePUnit`.
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean:300]: Do you mean "a diamond with"?

Comment (urkud) [reply Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean:300]:
```suggestion
- a diamond with the existing instance `MeasureTheory.Measure.instMeasureSpacePUnit`.
```

File: Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean [hunk 3]
```diff
@@ -290,7 +290,18 @@ end Fintype
 /-- A finite dimensional inner product space has a canonical measure, the Lebesgue measure giving
 volume `1` to the parallelepiped spanned by any orthonormal basis. We define the measure using
 some arbitrary choice of orthonormal basis. The fact that it works with any orthonormal basis
-is proved in `orthonormalBasis.volume_parallelepiped`. -/
+is proved in `orthonormalBasis.volume_parallelepiped`.
+
+This instance creates
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean:295]:
```suggestion
This instance creates:
```

**Review by eric-wieser — APPROVED** [review]

**Comment by eric-wieser** [comment]
@sgouezel, could you take a final look?

**Pull request closed without merging**